launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
Because it's being called from another package, which shouldn't have to know about yaml files in safety, and safety shouldn't have to look for yaml files in launch